services/system/ProxySys/src/ProxySys.cpp

```diff
@@ -79,10 +79,23 @@ namespace SystemService
       // TODO: avoid repacking (both directions)
       psibase::Actor<ServerInterface> iface(act.service, service);
 
+      auto reqTarget = req.target;
+
       auto result = iface.serveSys(std::move(req));
       if (result && !result->headers.empty() && serviceName != "common-sys")
          abortMessage("service " + service.str() + " attempted to set an http header");
 
+      if (reqTarget.size() == 0 || reqTarget.starts_with("/index.html"))
```

Comment:
Do you really mean `starts_with`? `ends_with` or `==` would make more sense to me.